### PR TITLE
travis-ci

### PR DESCRIPTION
**Pull request by daniel-vega-86**
## Summary

[Change!] add the file .travis.yml

## Screenshots

[Change!] N/A.

## Trello Card

[Change!] https://trello.com/c/aMrsEhR5/11-3-configurar-travisci

### CODE REVIEW

**Review by redwarewolf — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+before_install:
+  - "echo '--colour' > ~/.rspec"
+  - "echo 'gem: --no-document' > ~/.gemrc"
+  # Solves rainbow issue: https://github.com/sickill/rainbow/issues/48
+  - gem update --system
+
+before_script:
+  - cp config/database.travis.yml config/database.yml
+  - psql -c "CREATE USER \"rails-bootstrap\" WITH PASSWORD 'rails-bootstrap';" -U postgres
+  - psql -c "DROP DATABASE IF EXISTS \"rails-bootstrap_test\";" -U postgres
```
Comment (blacksam07) [inline .travis.yml:28]: this is another user and database 👀

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+addons:
+  code_climate:
+    repo_token: 89943dcf9eece00c7e360730a4a6102b1ce29c1b9956186bf4801ad059169274
+
+env:
+  global:
+    - RUBY_GC_MALLOC_LIMIT=90000000
+    - RUBY_GC_HEAP_FREE_SLOTS=200000
+    - SECURITY_SECRET_TOKEN=traviscitraviscitraviscitraviscitravisci
```
Comment (blacksam07) [inline .travis.yml:13]: 👀

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+services:
+  - postgresql
+  - redis-server
```
Comment (blacksam07) [inline .travis.yml:17]: you need redis here?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+language: ruby
+rvm:
+  - 2.6.5
```
Comment (blacksam07) [inline .travis.yml:3]: in your Gemfile have ruby version 3.0.1

**Review by blacksam07 — COMMENTED** [review]
@redwarewolf @daniel-vega-86  why this pr was merged?, Travis was never run and have inconsistent

**Pull request merged**